composer.libraries.json

```diff
@@ -1,307 +0,0 @@
-                "version": "1.13.18",
-                "type": "drupal-library",
-                "extra": {
-                    "installer-name": "jquery.timepicker"
-                },
-                "dist": {
-                    "url": "https://github.com/jonthornton/jquery-timepicker/archive/1.13.18.zip",
-                    "type": "zip"
-                },
-                "require": {
-                    "composer/installers": "~1.0"
-                }
-            }
-        },
-        "progress-tracker": {
-            "type": "package",
-            "package": {
-                "name": "progress-tracker/progress-tracker",
-                "version": "2.0.6",
-                "type": "drupal-library",
-                "extra": {
-                    "installer-name": "progress-tracker"
-                },
-                "dist": {
-                    "url": "https://github.com/NigelOToole/progress-tracker/archive/2.0.6.zip",
-                    "type": "zip"
-                },
-                "require": {
-                    "composer/installers": "~1.0"
-                }
-            }
-        },
-        "signature_pad": {
-            "type": "package",
-            "package": {
-                "name": "signature_pad/signature_pad",
-                "version": "2.3.0",
-                "type": "drupal-library",
-                "extra": {
-                    "installer-name": "signature_pad"
-                },
-                "dist": {
-                    "url": "https://github.com/szimek/signature_pad/archive/v2.3.0.zip",
-                    "type": "zip"
-                },
-                "require": {
-                    "composer/installers": "~1.0"
-                }
-            }
-        },
-        "tabby": {
-            "type": "package",
-            "package": {
-                "name": "tabby/tabby",
-                "version": "12.0.3",
-                "type": "drupal-library",
-                "extra": {
-                    "installer-name": "tabby"
-                },
-                "dist": {
-                    "url": "https://github.com/cferdinandi/tabby/archive/v12.0.3.zip",
-                    "type": "zip"
-                },
-                "require": {
-                    "composer/installers": "~1.0"
-                }
-            }
-        },
-        "tippyjs/6.x": {
-            "type": "package",
-            "package": {
-                "name": "tippyjs/6.x",
-                "version": "6.2.6",
-                "type": "drupal-library",
-                "extra": {
-                    "installer-name": "tippyjs/6.x"
-                },
-                "dist": {
-                    "url": "https://unpkg.com/tippy.js@6.2.6/dist/tippy-bundle.umd.min.js",
-                    "type": "file"
-                },
-                "require": {
-                    "composer/installers": "~1.0"
-                }
-            }
-        }
-    }
-}
```

Comment:
Jira ticket mentions this `composer.libraries.json` was moved into the `upstream-configuration/` folder, but I'm not seeing it there. Am I missing something?

Reply:
Good catch!